### ISSUE TITLE
If you shift click an item wile rolling you will take the item and you can roll forever

### ISSUE THREAD

**Issue by screamingcoder** [ConnorLinfoot/CratesPlus#504]
**Please use the issue template below. Issues not using the templates will likely be closed without warning! If you are suggesting a feature you do not need to follow the template.**

If you're having config issues (such as it resetting) first make sure it's a valid YAML file using a [parser](http://yaml-online-parser.appspot.com).

From version 4.3 there is a new '/crates debug' command. Running this will give you a single link with all the data I need to help you. Please run this and submit the share link with your issue.

### Debug Link        https://mcdebug.xyz/cratesplus/share/vJXbsZUA5b
___Enter your debug link from `/crates debug`. Not providing this may result in the issue being closed without warning. If you are unable to use this command please provide your Bukkit version, a list of plugins and your config.yml for CratesPlus.___

### Link/paste of stack trace/error (if any)

    You can also simply just use code formatting in markdown!
___We need all of the stack trace! Do not cut off parts of it.___

### Actions to reproduce (if known):
___This may include a build schematic, a video, or detailed instructions to help reconstruct the issue___

**Comment by Mosrod** [comment]
Yep, can reproduce. You can also press the number keys 1-10 for the item to go into the specified slot. The rolling again only happens during the rolling time. A solution is to use the NoGUI opener.

**Comment by Greenfoot5** [comment]
This happens whilst in creative for me, but, when you continue the items are wiped from your inventory, but, if you place the item then you keep it even after opening. If you open in survival mode, it closes the UI and removes the item. This was tested with the BasicGUI, I don't know about the CSGO opener.
